tests/integration/components/step-five-test.js

```diff
@@ -36,5 +36,9 @@ module('Integration | Component | step-five', function (hooks) {
       .hasText(
         'If the link is verified you will hear back with a personalized joining link within 7 working days'
       );
+    assert
+      .dom('[data-test-button="back-to-home"]')
+      .hasText('Back to Home')
+      .exists();
```

Comment:
Write an assert for redirection also

Reply:
added for route

Reply:
@sahsisunny We can test the button as follows :
- Check that current route is `/join`
- Click the button
- Check that current route is `/`

Reply:
added using acceptance